### ISSUE TITLE
Statistiques

### ISSUE THREAD

**Issue by Crali** [kevin-verschaeve/boite-a-beauf#1]
Pouvoir faire des stats sur la base des lectures des fichiers.
Par ex, un graphique qui montre les sons les plus joués.
- Incrémenter un compteur à la lecture d'un son
- Faire une page de stats
Utiliser une base sqlite ?

**Comment by kevin-verschaeve** [comment]
Ok c'est marrant mais ya un interet ?
retirer ceux qu'on joue le moins pour les remplacer ?

**Comment by Crali** [comment]
Yes, voir les sons les moins joués pour les retirer, voir les plus populaires etc
C'est + pour le fun
